### DISCUSSION TITLE
rke2  support

### DISCUSSION THREAD

**Discussion by AdZoAi** [mysticaltech/terraform-hcloud-kube-hetzner#801] [Ideas]
Still planning to add rke2 support in the next major release? any time frame for that?
Thanks!

**Comment by valkenburg-prevue-ch** [comment]
It is indeed still on top of the list of bigger things to take on. But I do not think there is a time frame. @ifeulner Is this correct?

**Reply by mysticaltech** [reply]
Indeed, we will get to it very soon.
